src/thermostate/plotting.py

```diff
@@ -75,6 +77,8 @@ def add_state(self, state, key=None):
             x_data = np.array(x_data) * getattr(units, self.axis_units[x_axis])
             y_data = np.array(y_data) * getattr(units, self.axis_units[y_axis])
             (line,) = axis.plot(x_data, y_data, marker="o")
+            if state._label is not None:
+                axis.annotate(state._label, (x_data[0], y_data[0]), textcoords = "offset pixels", xytext = (5, 5))
```

Comment:
On these two lines, you should use just `state.label` rather than `state._label`. The plotting doesn't need to know what variable the `state` actually stores the `label` in, just that it has some property called `label`. Also, you have some extra spaces around `=` here.